CONTRIBUTING.md

```diff
@@ -45,13 +45,13 @@ A proper header must be in place for any file contributed to Eclipse Kura. For a
  *
  *******************************************************************************/
  ```
- 
+
  Please ensure \<legal entity\> is replaced with the relevant information. If you are editing an existing contribution, feel free
  to create or add your legal entity to the contributors section as such:
- 
+
```

Comment:
if we mention <the current year>, we can also mention here that <the current year> should be replaced with the current year in numerics, for example, 2016 or 2017.